python/cuml/test/test_linear_model.py

```diff
@@ -21,80 +21,87 @@
 from cuml.test.utils import array_equal
 import cudf
 import numpy as np
+import pandas as pd
+from sklearn.datasets import make_regression
 
 
 @pytest.mark.parametrize('datatype', [np.float32, np.float64])
 @pytest.mark.parametrize('X_type', ['dataframe', 'ndarray'])
 @pytest.mark.parametrize('y_type', ['series', 'ndarray'])
 @pytest.mark.parametrize('algorithm', ['eig', 'svd'])
-def test_ols(datatype, X_type, y_type, algorithm):
-
-    X = np.array([[2.0, 5.0], [6.0, 9.0], [2.0, 2.0], [2.0, 3.0]],
-                 dtype=datatype)
-    y = np.dot(X, np.array([5.0, 10.0]).astype(datatype))
-
-    pred_data = np.array([[3.0, 5.0], [2.0, 5.0]]).astype(datatype)
-
+@pytest.mark.parametrize('nrows', [pytest.param(20, marks=pytest.mark.unit),
+                                   pytest.param(500000,
+                                                marks=pytest.mark.stress),
+                                   pytest.param(5000,
+                                                marks=pytest.mark.quality)])
+@pytest.mark.parametrize('ncols', [pytest.param(3, marks=pytest.mark.unit),
+                                   pytest.param(1000,
+                                                marks=pytest.mark.stress),
+                                   pytest.param(100,
+                                                marks=pytest.mark.quality)])
+@pytest.mark.parametrize('n_info', [pytest.param(2, marks=pytest.mark.unit),
+                                    pytest.param(500,
+                                                 marks=pytest.mark.stress),
+                                    pytest.param(50,
+                                                 marks=pytest.mark.quality)])
+def test_unit(datatype, X_type, y_type, algorithm, nrows, ncols, n_info):
```

Comment:
I think test_ols was a clearer name... if you want to suggest that it's a unit test, you could do test_ols_unit